internal/validation/testdata/tests.json

```diff
@@ -1752,17 +1873,7 @@
       "vars": {
         "colors": "BROWN"
       },
-      "errors": [
```

Comment:
This case was actually an example where input coercion should have _allowed_ the value, so expecting an error here was actually incorrect.